src/renderer/components/DeviceAction/rendering.js

```diff
@@ -251,6 +253,13 @@ export const renderError = ({
       <TranslatedError error={error} field="description" />
       <SupportLinkError error={error} />
     </ErrorDescription>
+    {list ? (
```

Comment:
can you check what happens if we don't even have an explicit `list` ? to me it's the expected behavior to ALWAYS enable this, what do you think?

Reply:
we're gonna have some weird CSS block problems if we do I believe

Reply:
what i mean is that we might want to enable the error every time there is a list in that error; the error you are refering to will probably happen were you are today using `list=true` btw. because other errors can happen like no network.

we need to change a bit some logic maybe to make it only having a `<ol>` when there is something inside. maybe that `<ol>` actually should be INSIDE the TranslatedError. I don't see a reason why we wouldn't do this. (maybe check usages of TranslatedError if we have other cases of using the list mode)

Reply:
let's review that later. good to merge for now.